src/objective-c/GRPCClient/private/GRPCConnectivityMonitor.h

```diff
@@ -72,6 +72,6 @@
  * Only one handler is active at a time, so if this method is called again before the previous
  * handler has been called, it might never be called at all (or yes, if it has already been queued).
  */
-- (void)handleLossWithHandler:(void (^)())lossHandler
-      wifiStatusChangeHandler:(void (^)())wifiStatusChangeHandler;
+- (void)handleLossWithHandler:(nullable void (^)())lossHandler
```

Comment:
From the [spiral rule of declarations](http://c-faq.com/decl/spiral.anderson.html), it would seem to me that this should be written:
```objective-c
(void (nullable ^)())lossHandler
```

But I wouldn't bet on it either! :) Give it a try, and see if the compiler doesn't complain?